assets/src/blocks/Media/MediaFrontend.js

```diff
@@ -0,0 +1,41 @@
+import { MediaEmbedPreview } from './MediaEmbedPreview';
+
+const { __ } = wp.i18n;
+
+const wrapAndRemoveSize = embedHtml => {
+  const wrapperDiv = document.createElement('div');
+  wrapperDiv.innerHTML = embedHtml;
+  if (wrapperDiv.firstChild && wrapperDiv.firstChild.src.indexOf('youtube') > -1) {
+    wrapperDiv.className = 'embed-container';
+    wrapperDiv.firstChild.removeAttribute('width');
+    wrapperDiv.firstChild.removeAttribute('height');
+    return wrapperDiv.outerHTML;
+  } else {
+    return embedHtml;
+  }
+}
+
+export const MediaFrontend = ({ attributes }) => {
+  const {
+    video_title,
+    description,
+    video_poster_img,
+    embedHtml,
+  } = attributes;
+
+  const wrappedHtml = wrapAndRemoveSize(embedHtml);
+
+  return (
+    <section className="block articles-block">
+      <div className="container">
+        <header>
```

Comment:
We should probably hide this if `video_title` is not defined, as we do below for `description` 🙂